### PR TITLE
Add README.md

### LINKED ISSUE #1: Add README

### PR DESCRIPTION
**Pull request by gustavosr8**

### CODE REVIEW

File: README.md
Comment (henriquesimoes) [inline README.md]: This is not quite right for this IOC. You should still define IP address and prefix for the PVs when launching the container and successfully connecting to a MCH.

File: README.md
Comment (henriquesimoes) [inline README.md:27]: Since you don't use `TAG` in your docker-compose.yml, there is no need to define it here.

**Review by henriquesimoes — CHANGES_REQUESTED** [review]

File: README.md
Comment (henriquesimoes) [inline README.md]: What about "EPICS IOC for uTCA crate management"?

File: README.md
Comment (henriquesimoes) [inline README.md]: Marking this as `shell` or `bash` can improve syntax highlight, I think.

File: README.md
Comment (henriquesimoes) [inline README.md]: `s/it's/its/`

File: README.md
Comment (ericonr) [inline README.md]:
```suggestion
This repository contains the EPICS Input/Output Controller (IOC) used at LNLS for monitoring and controlling MTCA crates,
based on the [ipmiComm](https://github.com/slac-epics-modules/ipmiComm) support module.
```

File: README.md
Comment (ericonr) [inline README.md]: If we decide in #14 to use tags, this will require it to be added.

**Review by ericonr — CHANGES_REQUESTED** [review]

File: README.md
Comment (gustavosr8) [reply README.md]: what about change `monitoring and controlling` for `managing`?

File: README.md
Comment (henriquesimoes) [inline README.md:4]: As I could understand, this will take the [default branch action status because we are not using `branch` parameter][1].

When we merge this, it won't trigger a workflow after merging, according to our CI rule. Will the badge status still be correct?

[1]: https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/monitoring-workflows/adding-a-workflow-status-badge#using-the-branch-parameter

Comment (gustavosr8) [reply README.md:4]: Hm, I guess that, in this case, as the `branch` parameter isn't being used, the badge will always shows the status of the last triggered CI workflow, so it will not display the status of the merging because it not triggers the workflow, as you notice, but will display the status of the last push or PR.

Anyway, that badge is almost a decoration, and I can remove it if the status could cause any misunderstandings.

**Review by henriquesimoes — APPROVED** [review]

**Review by ericonr — APPROVED** [review]

**Pull request merged**
